_opensearch/snapshots/searchable_snapshot.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: default
+title: Searchable snapshots
+parent: Snapshots
+nav_order: 40
+has_children: false
+---
+
+# Searchable snapshots
+
+Searchable snapshots is an experimental feature with OpenSearch 2.4. Therefore, we do not recommend the use of this feature in a production environment. For updates on progress or if you want leave feedback that could help improve the feature, see the [searchable snapshot GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/2919). 
+{: .warning }
+
+A searchable snapshot is an index where data is read from a [snapshot repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/#register-repository) on-demand at search time, rather than downloading all index data to cluster storage at restore time. Because the index data remains in the snapshot format in the repository, searchable snapshot indexes are inherently read-only. Any attempt to write to a searchable snapshot index will result in an error.
+
+To enable the searchable snapshots feature, reference the steps below.
+
+## Enabling the feature flag
+
+There are several methods for enabling searchable snapshots, depending on the install type.
+
+### Enable on a node using a tarball install
+
+The flag is toggled using a new jvm parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in config/jvm.options.
```

Comment:
"JVM"? Replace the ending period with a colon. Minor nitpick, but should we list the options here in the same order as they're presented below? Also, should Option 3 be referenced here as well?

Reply:
We're matching casing found in the actual filesystem. There was back and forth for option 3 in our last doc related to a feature like this and in the end we decided to keep it so I'm keeping that consistent here.